core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -199,14 +216,22 @@ public boolean releaseAllReaders() {
         return readerPool.releaseAll();
     }
 
-    public boolean releaseAllWriters() {
-        return writerPool.releaseAll();
-    }
-
-    public void releaseInactive() {
-        writerPool.releaseInactive();
-        readerPool.releaseInactive();
+    public boolean releaseAllWriters () {
+    	boolean released = writerPool.releaseAll();
+    	if (null != backupWriterPool && backupWriterPool.releaseAll()) {
+    		released = true;
+    	}
+    	return released;
     }
+    
+	public boolean releaseInactive() {
+		boolean useful = writerPool.releaseInactive();
+		useful |= readerPool.releaseInactive();
+		if (null != backupWriterPool) {
+			useful |= backupWriterPool.releaseInactive();
```

Comment:
code formatting is still inconsistent with rest of the code. This isn't the only line.